pycircleci/api.py

```diff
@@ -326,14 +326,14 @@ def get_project_pipelines(self, username, project, vcs_type=GITHUB, paginate=Fal
         :param project: Repo name.
         :param vcs_type: VCS type (github, bitbucket). Defaults to ``github``.
         :param paginate: If True, repeatedly requests more items from the endpoint until the limit has been reached (or until all results have been fetched). Defaults to False
-        :param limit: Maximum number of items to return in this response. By default returns all the results from many calls to the endpoint, or all the results from a single call to the endpoint, depending on the value for 'paginate'.
+        :param limit: Maximum number of items to return. By default returns all the results from multiple calls to the endpoint, or all the results from a single call to the endpoint, depending on the value for "paginate".
```

Comment:
Maybe not the style you prefer, but one other suggestion might be to wrap these longer parameter descriptions similar to the way described here?

https://stackoverflow.com/questions/32566466/multi-line-description-of-a-parameter-description-in-python-docstring